test/node/__preperation.js

```diff
@@ -0,0 +1,15 @@
+const Circuit = require('../../circuit-node');
+const config = require('./config.json');
+let client = new Circuit.Client(config.bot1);
+let user;
```

Comment:
No need to define user since its not used

Reply:
done